test/json_post_logger.ts

```diff
@@ -15,14 +15,8 @@ test('log via post request to example.com', async (done) => {
     { type: 'json_post', host: 'logging.example.com' },
   ]);
 
-  const loggingEndpoint = nock('http://logging.example.com')
-    .post('/', {
-      severity: 'log',
-      type: 'log',
-      message: 'Hello!',
-      info: {},
-      environment: 'test',
-    })
+  const loggingEndpoint = nock(/example.com/)
```

Comment:
instead of using /example.com/ can you use a bombbomb url for safety

Reply:
example is RFC'd for this purpose

Reply:
👍